stix2slider/test/test_idioms.py

```diff
@@ -137,7 +137,8 @@ def test_idiom_mapping(test_file, stored_master):
 
 
 def pytest_generate_tests(metafunc):
-    setup_tests()
+    version = os.environ['VERSION']
```

Comment:
Maybe we should use a more specific name for the environment variable. Also, we should configure Tox to set this variable with both versions to make sure we test everything.